### PR TITLE
Inital Recipe 

### PR DESCRIPTION
**Pull request by ritumd**
## Checklist before submitting

- [X] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

Fixes # (issue).

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,7 +32,8 @@ about:
   home: http://github.com/rspeer/langcodes
   summary: Labels and compares human languages in a standardized way
   license: MIT
-  license_file: LICENSE.txt
+  license_family: MIT
+  license_file: LICENS
```
Comment (alexlang74) [inline recipe/meta.yaml]: There is an `E` missing

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ source:
   sha256: 794d07d5a28781231ac335a1561b8442f8648ca07cd518310aeb45d6f0807ef6
 
 build:
-  number: 1
+  number: 3
```
Comment (cdeepali) [inline recipe/meta.yaml]: Should this be 2?

**Review by alexlang74 — APPROVED** [review]

**Review by cdeepali — APPROVED** [review]

File: recipe/meta.yaml [hunk 3]
```diff
@@ -33,7 +33,7 @@ about:
   summary: Labels and compares human languages in a standardized way
   license: MIT
   license_family: MIT
-  license_file: LICENSE
+  license_file: LICENSE.txt
```
Comment (alexlang74) [inline recipe/meta.yaml:36]: why is this `.txt` again? Isn#T the file you added called only `LICENSE` ?

Comment (ritumd) [reply recipe/meta.yaml:36]: Hi @alexlang74 , This License file points to source code license and not to openCE one

Comment (cdeepali) [reply recipe/meta.yaml:36]: Yes, the license_file name here has to match the file name in the source code - https://github.com/rspeer/langcodes/blob/master/LICENSE.txt

**Review by cdeepali — APPROVED** [review]

**Pull request merged**